Src/PerformanceCollector/Unit.Tests/QuickPulse/QuickPulseServiceClientTests.cs

```diff
@@ -23,7 +23,7 @@ public sealed class QuickPulseServiceClientTests : IDisposable
 
         private readonly List<Tuple<DateTimeOffset, MonitoringDataPoint>> samples = new List<Tuple<DateTimeOffset, MonitoringDataPoint>>();
 
-        private readonly List<Tuple<DateTimeOffset, MonitoringDataPoint>> pings = new List<Tuple<DateTimeOffset, MonitoringDataPoint>>();
+        private readonly List<Tuple<PingHeaders, MonitoringDataPoint>> pings = new List<Tuple<PingHeaders, MonitoringDataPoint>>();
```

Comment:
Are those headers changes much? Could they be set once in constructor or initialization of sender?